StreamChatUITestsAppUITests/Robots/ParticipantRobot.swift

```diff
@@ -59,10 +59,10 @@ final class ParticipantRobot: Robot {
         return self
     }
     
-    // Think twice before using it
+    // Sleep in milliseconds
     @discardableResult
-    func chill(duration: UInt32) -> Self {
-        sleep(duration)
+    func chill(duration: useconds_t) -> Self {
```

Comment:
Can we please use `TimeInterval` here and then map to milliseconds? The common pattern in iOS is to use seconds (TimeInterval) as the unit of time

Reply:
Another possible implementation is like this:

```swift
func wait(timeout: TimeInterval) {
        let runLoop = RunLoop.current
        runLoop.run(until: Date(timeIntervalSinceNow: timeout))
    }
```